### ISSUE TITLE
error: [Errno 48] Address already in use

### ISSUE THREAD

**Issue by jetzhao04** [idpaterson/alfred-wunderlist-workflow#83]
Hi, I am stuck in the login step. Sounds like localhost (http://localhost/?state=DJTF97QLI54PTAJGROPP&code=251bdebafa1bed13c1e3) is not listening, or port in use?

Here is the debug info: 

```
Starting debug for 'Wunderlist'

[STDERR: alfred.workflow.input.scriptfilter] 10:08:12 workflow.py:1498 DEBUG    No data stored for `prefs`
10:08:12 workflow.py:1974 DEBUG    Workflow version : 0.4.1
10:08:12 workflow.py:1389 DEBUG    Reading settings from `/Users/xxx/Library/Application Support/Alfred 2/Workflow Data/com.ipaterson.alfred.wunderlist/settings.json` ...
10:08:12 workflow.py:2235 DEBUG    Update check not due
10:08:12 workflow.py:1612 DEBUG    Loading cached data from : /Users/xxx/Library/Caches/com.runningwithcrayons.Alfred-2/Workflow Data/com.ipaterson.alfred.wunderlist/__workflow_update_status.cpickle
10:08:12 workflow.py:2183 DEBUG    update_data : {u'available': False}
10:08:12 workflow.py:2165 DEBUG    Set last run version : 0.4.1
10:08:12 workflow.py:2010 DEBUG    Workflow finished in 0.039 seconds.
[ERROR: alfred.workflow.action.script] 10:08:13 workflow.py:1498 DEBUG    No data stored for `prefs`
10:08:13 workflow.py:1974 DEBUG    Workflow version : 0.4.1
10:08:13 workflow.py:1389 DEBUG    Reading settings from `/Users/xxx/Library/Application Support/Alfred 2/Workflow Data/com.ipaterson.alfred.wunderlist/settings.json` ...
10:08:13 workflow.py:2235 DEBUG    Update check not due
10:08:13 workflow.py:2305 DEBUG    Password exists : com.ipaterson.alfred.wunderlist:oauth_state
10:08:13 workflow.py:2352 DEBUG    Got password : com.ipaterson.alfred.wunderlist:oauth_state
10:08:13 workflow.py:2374 DEBUG    Deleted password : com.ipaterson.alfred.wunderlist:oauth_state
10:08:13 workflow.py:2315 DEBUG    save_password : com.ipaterson.alfred.wunderlist:oauth_state
Process Process-1:
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/xxx/Library/Application Support/Alfred 2/Alfred.alfredpreferences/workflows/user.workflow.DF475A1D-2F27-40A5-83F2-9CE2C07A9514/wunderlist/auth.py", line 112, in await_token
    server = SocketServer.TCPServer(("", config.OAUTH_PORT), OAuthTokenResponseHandler)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/SocketServer.py", line 420, in __init__
    self.server_bind()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/SocketServer.py", line 434, in server_bind
    self.socket.bind(self.server_address)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 48] Address already in use
10:08:13 workflow.py:2165 DEBUG    Set last run version : 0.4.1
10:08:13 workflow.py:2010 DEBUG    Workflow finished in 0.269 seconds.
```

**Comment by jetzhao04** [comment]
Saw the #82 issue by @idpaterson , but http://localhost:6200/?state=E1MJN9O0KP5T8DR6RC65&code=2f1261857a31515d4888 doesn't work for me either.

**Comment by idpaterson** [comment]
Do you know if this turned out to be another service running on port 6200 or were there two instances of the workflow authentication server running?

I would love to be able to specify a range of ports in the API so that an alternate could be used if 6200 is not available (#70 addresses this particular issue)
